tasks/Advertisement Writing.md

```diff
@@ -0,0 +1,54 @@
+consists of members who have never interacted before in groups of 3-4 people 
+> 2. Participants play four rounds of that task (order balanced across groups: 2 with same group, two with new group). Teams wrote
+ads for different Kickstarter projects in each round, so they would never repeat the exact same task. At the end of the round, the team collectively
+decided on a final advertisement and one member submitted it on behalf of the team.
+> 3. At the end of each round participants complete a viability scale
+> 4. At the end of all rounds participants attitudinal data collected 
+
+
+## Instructions
+Collaborate with your team members in the chat room to write an ad! Check out the link below to view the product! This task has 5 rounds that last ten minutes, followed up a 5 minute survey. You will be bonused at a rate of $12/hr.
+
+Example product link: https://www.kickstarter.com/projects/1636469325/the-tis-tea-plant-based-high-rich-minerals-in-seaw
+
+The following instructions appear during the course of the activity:
+1. As a team, brainstorm as many ad slogans as possible and then narrow down your ideas to determine your final ad submission.
+2. Your final ad will appear online.
+3. The more successful it is, the larger the bonus each team member will be paid.
+4. The ad must be no more than 30 characters long. Instructions for submission will appear later.
+5. There are a total of 5 rounds of team interaction with a 5 minute reflection survey following each one. You will only receive the stated bonus pay if you thoughtfully fill out every survey question.
+6. Your ads will be reviewed and checked after each task. If you go AFK on your teammates, you will NOT receive the stated pay.
+7. Say hello to your team members. Begin brainstorming ideas together!
+8. As a team, discuss and narrow down your ideas!
+9. Polish your team’s favorite ad and get ready to submit. Submit your ad by adding a “!” in front of the ad. For example, “!Ad text here.” Remember, your final ad can’t be more than a maximum of 30 characters long.
+10. This round is ending soon! Say goodbye to your teammates.
+
+# Criteria
+## Performance calculation
+Team fracture: 
+> Associated with low average team viability scores:  at the end of each round, each participant filled out a 14-item viability scale drawn from prior work
+> Attitudinal data: when at least half of a group’s members privately excluded that group from future rounds 
+
+Ad Performance: 
+For the creative ad writing task, each advertisement was added to a campaign for the product on Google Ads. We turned off any optimization, ensuring that ads were seen at the same rate. After 48 hours, we recorded each ad’s click through rate (CTR) as a measure of the group’s performance
+
+## Incentives
+Participants were paid $12 per hour for taking part in the experiment, and were incentivized to complete the entire study by receiving a majority of their payment as a bonus at the end of the experiment.
```

Comment:
Maybe mention that the $2 was only rewarded for above average work.
"We paid $10 per task and each task took at most 1
hour to complete. We also offered a bonus of up to $2 for
above average work."